### DISCUSSION TITLE
Is it safe to use with any Linux distro? Will it mess with system installed Ruby?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [rbenv/ruby-build#2410] [General]
Hi, I use Gentoo by the way, but 3.3 is still missing from the system package manager.

**Comment by vitaly-zdanevich** [comment]
And what is the correct way to uninstall?

**Comment by mislav** [comment]
Hi, ruby-build is a program that when invoked as:

```
ruby-build 3.3.2 /path/to/ruby
```

will install a Ruby 3.3 into the specified prefix `/path/to/ruby`. This will not mess up your system Ruby. You do not need a specific distro to use ruby-build, as this program will work on any non-Windows system that has bash and compiler utilities installed.

The way to uninstall a Ruby version installed this way is to `rm -rf /path/to/ruby`. The way to uninstall ruby-build is to remove the files you downloaded from this repository (e.g. `rm -rf /path/to/ruby-build`).
